products/knownprojects/bash/03_aggregate.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+source ../../bash/utils.sh
+set_error_traps
+
+echo "Preprocess for all aggregations"
+run_sql_command "ALTER TABLE _kpdb RENAME COLUMN geom TO geometry;"
+
+echo "Create ZAP Project Many BBLs table"
+run_sql_file aggregate/create_zap_projects.sql
+
+## Do SCA aggregation
+echo "Create the longfrom SCA Aggregate Tables..."
+
+# Preprocess the tables to standardize geometry column name 
+echo "Preprocess column names to standardize"
+run_sql_file aggregate/sca/preprocessing.sql
```

Comment:
is there a big reason for the sca subfolder at this point? If these are all just similar aggregations by different geographies

Reply:
the only reasons I can think of were
- to make it a little easier to test the new files. since these all take so long, disabling all SCA aggregation was easier when they're in a subfolder and have their own [`preprocessing.sql`](https://github.com/NYCPlanning/data-engineering/blob/kpdb-geographies/products/knownprojects/aggregate/sca/preprocessing.sql)
- unlike the new files, these are kinda more specific and are related to each other because they're meant to be shared with the SCA
- per that last one, they're exported and [zipped](https://github.com/NYCPlanning/data-engineering/blob/kpdb-geographies/products/knownprojects/bash/04_export.sh#L61-L62) separate from the new, more general, aggregations